qpc/utils.py

```diff
@@ -352,7 +352,11 @@ def setup_logging(verbosity):
     # those not coming from qpc, will go to the log file
     logging.basicConfig(filename=QPC_LOG, format=log_fmt, level=log_level)
     stream_handler = logging.StreamHandler()
-    stream_handler.setFormatter(logging.Formatter(log_fmt))
+    if log_level == logging.DEBUG:
+        # changing log format was breaking camayoc tests. let's add this extra logging
+        # information only when the user actually requests for more logs.
+        # (at least until we add an option controlling the log format)
```

Comment:
Will we add this info to documentation as well, so the user knows he has this option?

Reply:
We can point the user can pass -v or -vv for more verbosity on any command if its not already mentioned.

What do you think @justinkreft ?

Reply:
But honestly I expect any cli user to run `-h`, `--help` or `man <COMMAND>` when looking for instructions.

Reply:
Fair point